kiva/fonttools/tests/test_scan_parse.py

```diff
@@ -42,8 +42,9 @@ def test_fontlist_from_ttc(self):
 
         # Then
         self.assertEqual(len(fontlist), 2)
-        for fontprop in fontlist:
+        for idx, fontprop in enumerate(fontlist):
             self.assertIsInstance(fontprop, FontEntry)
+            self.assertEqual(fontprop.face_index, idx)
```

Comment:
@rahulporuri Does this count as a test for `face_index`?

Reply:
definitely. resolving!